thirdeye/thirdeye-frontend/app/utils/yaml-tools.js

```diff
@@ -29,6 +28,20 @@ rules:
     type: DATA_SLA              # Alert if data is missing.
     params:
       sla: 3_DAYS               # Data is missing for 3 days since last availability
+
+# You can mention a cron to allow this detection to be run periodically
+# For example, the cron below would execute every 5 minutes.
+# cron: "0 0/5 * 1/1 * ? *"
+
+# Detections in past data
+# You can use the lastTimestamp parameter to denote how far back you want the anomalies
+# to be detected. This value acts like a checkpoint or high watermark. 
+# A valid entry is a non negative timestamp value in millis.
+# 
+# - By default, (if not mentioned), the lookback is 30 days
+# - A value of 0 (zero) implies complete backfill till the start of data. As shown below.
```

Comment:
Couple of concerns with backfilling all the data:
1. Backfilling all the data can take quite some time, especially if there are years worth of data and possibly even fail due to timeout and other issues.
2. There isn't much benefit of backfilling older data. Users usually don't care about historical anomalies/issues, at least not older than a month.
3. The goal of the 30 day default backfill is to quickly show some stats and recent anomalies as soon as the alert is created. Otherwise, the alert page will look empty and users have to revisit after some time.
4. Typically, users tend to create an alert using the basic settings and update it later on. When they update,  anomalies created using the initial setting will no longer make sense.

We would increase the default limit from 30 days to say 90 days if that is more reasonable?